### PR TITLE
Explain the per world settings

### PR DESCRIPTION
**Pull request by jacksyyy**
there is issue with MANY people not knowing about how to use the per world settings. the PurpurDocs do not explain it. they just say default. so I add more! this is simple to follow with example

### CODE REVIEW

File: mkdocs/Configuration.md [hunk 1]
```diff
@@ -385,7 +385,68 @@ Check out https://minecraft.fandom.com/wiki/Custom_world_generation#Structure_de
...
+  default:
+    hunger:
+      starvation-damage: 1.0 # This is auto generated
+  world_nether:
+    hunger:
+      starvation-damage: 0.5
+  world_the_end:
+    hunger:
+      starvation-damage: 2.0
+```
```
Comment (BillyGalbreath) [inline mkdocs/Configuration.md:406]: spawn chunks or starvation damage? ;)

**Review by BillyGalbreath — CHANGES_REQUESTED** [review]

**Comment by BillyGalbreath** [comment]
We (the bukkit communities) have generally been passing around this pdf: https://www.spigotmc.org/attachments/per-world-guide-pdf.444348/

**Comment by jacksyyy** [comment]
I can not view pdf because it requires the spigotmc account that I do not have. I will make one to view, but I think having included in docs is good idea anyways. PDF is format of the 2000s. it is 2022. I think it is possible many server owner are not used to PDF.

**Comment by granny** [comment]
I noticed that this explanation is nearly identical to Paper's; https://docs.papermc.io/paper/per-world-configuration

Decided to just link readers to their page. This means I won't have to worry about inconsistencies between our pages or have to keep something updated when there's hardly any difference between the two pages.

Appreciate the PR nonetheless!

**Pull request closed without merging**